lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -227,6 +229,42 @@
                     return this.indicators.success + gettext('Successfully unlinked.');
                 }
             }),
+            
+            OrderHistoryFieldView: FieldViews.ReadonlyFieldView.extend({
+                fieldType: 'orderHistory',
+                fieldTemplate: field_order_history_template,
+                events: {
+                    'mouseenter .u-field-order': 'showDetailsButton',
+                    'mouseleave .u-field-order': 'hideDetailsButton'
+                },
+
+                initialize: function (options) {
+                    this.options = options;
+                    this._super(options);
+                    _.bindAll(this, 'showDetailsButton', 'hideDetailsButton');
+                },
+
+                render: function () {
+                    this.$el.html(this.template({
```

Comment:
Need to use the updated safe templating approach ```HtmlUtils.setHtml()``` http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#safe-javascript-files